core-foundation/src/url.rs

```diff
@@ -75,15 +74,20 @@ impl CFURL {
 
     #[cfg(unix)]
     pub fn to_path(&self) -> Option<PathBuf> {
+        // https://github.com/rust-lang/libc/blob/9d4e7802112e3d3186bdd0191a036281506c28b7/src/unix/linux_like/mod.rs#L1026
+        pub const PATH_MAX: c_int = 4096;
```

Comment:
PATH_MAX is 1024 on macOS

Reply:
You're right, I didn't check this properly; have brought back `libc` for this, but have made it possible to disable under a feature flag instead.